express-post-json/index.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const app = express();
+
+const grades = [];
+
+const getJson = app.get('/api/grades', (req, res) => {
+  res.json(grades);
+});
+
+const jsonParse = express.json(getJson);
+app.use(jsonParse, (req, res) => {
+  grades.push(req.body);
+  res.sendStatus(201);
```

Comment:
Each new grade object should receive a new id property. The server should respond with the created object too.

Reply:
Express Post JSON Req Res Updates | Yang An Boada
9c1c63e